src/powerhouse/components/modal/modal.tsx

```diff
@@ -36,6 +45,9 @@ export function Modal(props: Props) {
                             contentProps?.className,
                         )}
                     >
+                        <VisuallyHidden.Root>
```

Comment:
Fixes the annoying warning when a dialog is opened. We should probably make `title` required for accessibility.